### PR TITLE
nikto: add page

### PR DESCRIPTION
**Pull request by Dalmau**
----

- [X ] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X ] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/nikto.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# nikto
+
+> Web server scanner which performs comprehensive tests against web servers for multiple items, including over 6700 potentially dangerous files/programs.
+
+- To update to the latest plugins and databases:
```
Comment (sbrl) [inline pages/common/nikto.md]: Perhaps `Update to the`... would read better?

File: pages/common/nikto.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`perl nikto.pl -update`
+
+- The most basic Nikto scan requires simply a host to target:
```
Comment (sbrl) [inline pages/common/nikto.md]: Perhaps `Perform a basic Nikto scan against a target host` would read better?

File: pages/common/nikto.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`perl nikto.pl -h 192.168.0.1`
+
+- To check on a different port:
```
Comment (sbrl) [inline pages/common/nikto.md]: Perhaps `Specify the port number when performing a basic scan` would read better?

**Comment by Dalmau** [comment]
Hi @sbrl , I' ve put the news change. Thanks for check !!

File: pages/common/nikto.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# nikto
+
+> Web server scanner which performs comprehensive tests against web servers for multiple items, including over 6700 potentially dangerous files/programs.
+
+- Update to to the latest plugins and databases:
```
Comment (agnivade) [inline pages/common/nikto.md]: Make this the last example.

File: pages/common/nikto.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# nikto
+
+> Web server scanner which performs comprehensive tests against web servers for multiple items, including over 6700 potentially dangerous files/programs.
```
Comment (agnivade) [inline pages/common/nikto.md]: Let us remove the last part of the sentence. It is a bit unnecessary for a heading to contain such extraneous info.

File: pages/common/nikto.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+- Scan ports and protocols with full URL syntax:
+
+`perl nikto.pl -h https://192.168.0.1:443/`
+
+- Scan multiple ports in the same scanning session:
+
+`perl nikto.pl -h 192.168.0.1 -p 80,88,443`
+
+- Set the proxy on the command line:
```
Comment (agnivade) [inline pages/common/nikto.md]: This is not that commonly used. Let us remove it.

**Comment by Dalmau** [comment]
@agnivade, Thanks !!

**Pull request merged**